packages/pharos/src/components/tabs/pharos-tabs.test.ts

```diff
@@ -244,4 +244,27 @@ describe('pharos-tabs', () => {
     await component.updateComplete;
     expect(count).to.equal(0);
   });
+
+  it('does not render horizontal line', async () => {
+    const spanElement = component.renderRoot.querySelector('span') as HTMLDivElement;
+
+    expect(spanElement).to.be.null;
+  });
+
+  it('renders horizontal line', async () => {
+    component = await fixture(html`
+      <pharos-tabs horizontalline="true">
```

Comment:
It feels like e.g. `panel-separator` might provide more clarity here. "Horizontal line" leaves a fair amount to the imagination!